src/main/java/org/apache/datasketches/hive/cpc/UnionSketchUDF.java

```diff
@@ -24,12 +24,22 @@
 import org.apache.datasketches.cpc.CpcSketch;
 import org.apache.datasketches.cpc.CpcUnion;
 import org.apache.datasketches.hive.common.BytesWritableHelper;
+import org.apache.hadoop.hive.ql.exec.Description;
 import org.apache.hadoop.hive.ql.exec.UDF;
 import org.apache.hadoop.io.BytesWritable;
 
 /**
  * Hive union sketch UDF.
  */
+@Description(
+    name = "unionSketch",
+    value = "_FUNC_(firstSketch, secondSketch[, lgK[, seed]]) - Compute the union of the given "
+        + "sketches with the given size and seed",
+    extended = "The return value is a binary blob that can be operated on by other sketch related functions."
+        + " The lgK parameter controls the sketch size and rlative error expected from the sketch."
```

Comment:
typo "rlative"

Reply:
Thanks. I fixed all typos. Please let me know if you prefer to have separate PRs.
https://github.com/apache/datasketches-hive/pull/66/commits/e3bafbe37aecd3566369676299cf1c5437374d54

```
$ git grep rlative
src/main/java/org/apache/datasketches/hive/cpc/DataToSketchUDAF.java:    + " The lgK parameter controls the sketch size and rlative error expected from the sketch."
src/main/java/org/apache/datasketches/hive/cpc/UnionSketchUDAF.java:    + " The lgK parameter controls the sketch size and rlative error expected from the sketch."
src/main/java/org/apache/datasketches/hive/cpc/UnionSketchUDF.java:        + " The lgK parameter controls the sketch size and rlative error expected from the sketch."
src/main/java/org/apache/datasketches/hive/hll/DataToSketchUDAF.java:    + " The lgK parameter controls the sketch size and rlative error expected from the sketch."
src/main/java/org/apache/datasketches/hive/hll/UnionSketchUDAF.java:    + " The lgK parameter controls the sketch size and rlative error expected from the sketch."
src/main/java/org/apache/datasketches/hive/hll/UnionSketchUDF.java:        + " The lgK parameter controls the sketch size and rlative error expected from the sketch."
```